src/main.cpp

```diff
@@ -7,6 +7,8 @@
 #include "oscaroutput.h"
 #include "params.h"
 #include "tree.h"
+/* build dependent variables */
```

Comment:
I would remove this comment. Looking at _config.h_ it is clear what it is about.